### PR TITLE
L1T muon offline DQM config cleaning - 93x

### PR DESCRIPTION
**Pull request by thomreis**
Cleaning up of L1T muon offline DQM configuration

- Renaming of config files to be in line with Egamma naming scheme
- Changing the directory for the DQM output histograms
- Using the common efficiency harvesting module also used by L1TEgamma and dropping the special muon one
- Adding axis titles to efficiency plots
- Renaming the L1T muon offline module since it will not only produce histograms for efficiencies in the future but also resolution histograms and others.

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by dmitrijus** [comment]
+1

File: DQMOffline/L1Trigger/interface/L1TMuonOffline.h [hunk 1]
```diff
@@ -110,10 +110,10 @@ class MuonGmtPair {
 // DQM class declaration
 //
 
-class L1TEfficiencyMuons_Offline : public DQMEDAnalyzer {
+class L1TMuonOffline : public DQMEDAnalyzer {
```
Comment (davidlange6) [inline DQMOffline/L1Trigger/interface/L1TMuonOffline.h]: hi @thomreis - while we are moving the name of this around- it ought to include DQM somewhere in its name. Thanks

**Comment by thomreis** [comment]
Hi @davidlange6 I have added DQM to the file name and the module name.

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**